### ISSUE TITLE
Error 500 on calendar date options

### ISSUE THREAD

**Issue by citystrolch** [lochmueller/calendarize#587]
I have tried to grant access to a backend user / group, so they can add / edit events. The only issue so far is the calendar dates. Even though I have allowed all fields of the table, allowed type "events" in the edit and list tables and have allowed Calendar no Page Content: Plugin Section, the user gets an error 500 when trying to create new calendar dates.
I don't know if it's an issue or just misconfiguration, but I can't see any other options. 

Can any one provide me with some hints?

**Comment by okmiim** [comment]
Could you try to grant access to `Configuration` and `Configuration Group`?

**Comment by lochmueller** [comment]
Hey @citystrolch 
Do you have the error message? If you saw the 500 error in the backend (small red box), you saw the error in the AJAX call history of your browser? Furthermore, I suggest checking the rights as @okmiim  already mentioned.
Regards,
Tim

**Comment by lochmueller** [comment]
Hmm... currently no more information. Please create a new issue with the error message of the 500 error, if the problem still exists. So we can check, what the problem is in detail...

Thank you,
Tim

**Comment by citystrolch** [comment]
Hi everyone - sorry I was away and unable to respond. I wil check your kind suggestions, thank you for all the help. If there still is a problem, I will open a new issue.

**Comment by citystrolch** [comment]
Hello again, finally I got to work on this and I will keep working on it until solved now. I managed to get AJAX Error informations. This what creates an error: /typo3/index.php?route=/ajax/record/inline/create&token=... (token removed)
This is the response: No table modify permission for user 18 on table tx_calendarize_domain_model_configuration 

I just wanted to say that I think I managed. I had to give Configuration access and also enable all the fields in the configuration section of the user group (like start time, frequency etc.)

I am missing all this information in the documentation, as I find the group configuration absolutely not straight forward. Probably due to the great flexibility of the extension.

Kind regards and thank you for pointing me into the right direction.
Terry
